src/interfaces/highs_c_api.h

```diff
@@ -211,80 +213,80 @@ void* Highs_create(void);
  *
  * To empty a model without invalidating `highs`, see `Highs_clearModel`.
  *
- * @param highs     a pointer to the Highs instance
+ * @param highs     A pointer to the Highs instance.
  */
 void Highs_destroy(void* highs);
 
 /**
- * Return the HiGHS version number vX.Y.Z
+ * Return the HiGHS version number as a string of the form "vX.Y.Z".
  *
- * @returns the HiGHS version as a char*
+ * @returns The HiGHS version as a `char*`.
  */
 const char* Highs_version(void);
 
 /**
- * Return the HiGHS major version number
+ * Return the HiGHS major version number.
  *
- * @returns the HiGHS major version number
+ * @returns The HiGHS major version number.
  */
-HighsInt Highs_versionMajor();
+HighsInt Highs_versionMajor(void);
```

Comment:
Should I make this a separate PR? I got a warning parsing this in Julia.

Reply:
No need for a separate PR. I've no idea what this `void` does, anyway